salt/salt.spec

```diff
@@ -1710,7 +1706,26 @@ rm -f %{_localstatedir}/cache/salt/minion/thin/version
 %python_alternative %{_exec_prefix}/libexec/salt/salt-run
 %python_alternative %{_exec_prefix}/libexec/salt/salt-ssh
 %python_alternative %{_exec_prefix}/libexec/salt/salt-syndic
+%python_alternative %{_exec_prefix}/libexec/salt/zyppnotify
+%endif
+
+# Install plugin only on SUSE machines
+%if 0%{?suse_version}
+%{_prefix}/lib/zypp/plugins/commit/zyppnotify
```

Comment:
`zyppnotify` is the symlink here and should be a part of `salt-minion`, but it's listed here as a part of main package, so it's duplicated within all `python3xx-salt` most probably will cause conflicts in case of installing more than one of the flavours.

Reply:
You are right. I didn't show errors though when installing more than one flavor.

In any case, it makes sense to move the symlink to the Salt Minion package. I'll change this.

Reply:
This should be fixed now.